holoviews/core/util.py

```diff
@@ -729,12 +729,17 @@ def get_spec(obj):
 
 def find_file(folder, filename):
     """
-    Find a file given folder and filename.
+    Find a file given folder and filename. If the filename can be 
+    resolved directly 
     """
     matches = []
+    if os.path.isabs(filename) and os.path.isfile(filename):
+        return filename
     for root, _, filenames in os.walk(folder):
         for filename in fnmatch.filter(filenames, filename):
             matches.append(os.path.join(root, filename))
```

Comment:
Might be worth avoid overwriting the ``filename`` argument with the return values of ``fnmatch.filter`` (i.e it might be worth calling it ``fname`` in this loop).